docs/data/date-pickers/timezone/timezone.md

```diff
@@ -0,0 +1,387 @@
+
+function App() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterMoment}>
+      <DateTimePicker defaultValue={moment.utc('2022-04-17T15:30')} />
+    </LocalizationProvider>
+  );
+}
+```
+
+{{"demo": "MomentUTC.js", "defaultCodeOpen": false}}
+
+### Moment and timezone
+
+Before using the timezone with Day.js, you have to pass the default export from `moment-timezone` to the `dateLibInstance` prop of `LocalizationProvider`:
+
+```tsx
+import moment from 'moment-timezone';
+
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+
+<LocalizationProvider dateAdapter={AdapterMoment} dateLibInstance={moment}>
+  {children}
+</LocalizationProvider>;
+```
+
+:::info
+**How to create a date in a specific timezone with Moment?**
+
+If your whole application is using dates from the same timezone, set the default zone to your timezone name:
+
+```tsx
+import moment from 'moment-timezone';
+
+moment.tz.setDefault('America/New_York');
+
+const date = moment('2022-04-17T15:30');
+```
+
+If you only want to use dates with this timezone on some parts of your application, create a date using the `moment.tz` method:
+
+```tsx
+import moment from 'moment-timezone';
+
+const date = moment.tz('2022-04-17T15:30', 'America/New_York');
+```
+
+You can check out the documentation of the [timezone on Day.js](https://day.js.org/docs/en/timezone/timezone) for more details.
+:::
+
+You can then pass your date in the wanted timezone to your picker:
+
+```tsx
+import moment from 'moment-timezone';
+
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+
+function App() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterMoment} dateLibInstance={moment}>
+      <DateTimePicker
+        defaultValue={moment.tz('2022-04-17T15:30', 'America/New_York')}
+      />
+    </LocalizationProvider>
+  );
+}
+```
+
+{{"demo": "MomentTimezone.js", "defaultCodeOpen": false}}
+
+:::info
+You can check out the documentation of [Moment timezone](https://momentjs.com/timezone/) for more details on how to manipulate the timezones.
+:::
+
+## More advanced examples
+
+:::info
+The following examples are all built using `dayjs`.
+You can achieve the exact same behavior using `luxon` or `moment`,
+please refer to the sections above to know how to pass a UTC date or a date in a specific timezone to your component.
+:::
+
+### Store UTC dates but display in system's timezone
+
+The demo below shows how to store dates in UTC while displaying using the system timezone.
+
+{{"demo": "StoreUTCButDisplaySystemTimezone.js",  "defaultCodeOpen": false}}
+
+### Store UTC dates but display in another timezone
+
+The demo below shows how to store dates in UTC while displaying using the `America/New_York` timezone.
```

Comment:
I can take a more exotic timezone to lower the amount of people with the same timezone.

But adding a different timezone specifically for those people will be hard, because of the static generation of the doc

Reply:
`Pacific/Honolulu` Sounds great!

Reply:
Modified :+1: